react-native.config.js

```diff
@@ -26,10 +26,5 @@ module.exports = {
                 ios: null
             }
         }
-    },
-    project: {
```

Comment:
Is RN now able to find the project correctly on its own?

Reply:
project.ios.project is Deprecated and removed, instead of this there is new [project.ios.sourceDir](https://github.com/react-native-community/cli/blob/main/docs/projects.md#how-does-it-work) config. I tried something like `.ios` but didn’t work. But I  run debug mode with no problem